src/reducers/platform.js

```diff
@@ -71,16 +73,9 @@ export function clickTracking(state=DEFAULT_CLICK_TRACK, action) {
 }
 
 export function config(state={}, action) {
-	let apiUrl,
-		trackId;
-
 	switch(action.type) {
 	case 'CONFIGURE_API_URL':
-		apiUrl = action.payload;
-		return { ...state, apiUrl };
-	case 'CONFIGURE_TRACKING_ID':
```

Comment:
this is dead code cleanup - tracking ids are managed entirely by cookies, not Redux